### DISCUSSION TITLE
Week Field: Start or End?

### DISCUSSION THREAD

**Discussion by CarloGraziani** [hodcroftlab/covariants#212] [General]
Hello.

(1) Thanks for assembling and curating this data. It's crucial to something we're working on (we will follow the citation guidelines described in your FAQ of course).

(2) In the file cluster_tables/USAClusters_data.json, does the "week" array refer to the beginning of the reporting period, or to the end? In the case of the delta variant, the advent of prevalence in the USA was so rapid that it actually can matter quite a bit which interpretation is correct.

Cheers,

Carlo

**Comment by emmahodcroft** [comment]
Answered in issue #213
(Short answer is: the start!)
